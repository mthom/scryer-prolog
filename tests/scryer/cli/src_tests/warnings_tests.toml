args = [
    "-f",
    "--no-add-history",
    "src/tests/warnings.pl",
    "src/tests/warnings1.pl",
    "src/tests/warnings2.pl",
    "src/tests/warnings3.pl",
    "-g", "halt"
]
