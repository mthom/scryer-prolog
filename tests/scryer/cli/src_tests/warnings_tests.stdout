   error(permission_error(modify,static_procedure,(',')/2),load/1).
   error(permission_error(modify,static_procedure,(/+)/1),load/1).
   error(type_error(evaluable,'.'/2),load/1).
